src/directory-entry.js

```diff
@@ -2,5 +2,5 @@ var NODE_TYPE_FILE = require('./constants.js').NODE_TYPE_FILE;
 
 module.exports = function DirectoryEntry(id, type) {
   this.id = id;
-  this.type = type || NODE_TYPE_FILE;
+  this.mode = type || NODE_TYPE_FILE;
```

Comment:
This was previously `mode`, but changed to `type` in an earlier PR, is that right?

That change was not bundled into a release though?

Reply:
That is correct.  I did it during the summer when I was writing the Plan 9 stuff with Filer.  We never shipped it to `npm` or offered built versions, so it's fine to revert it now and we won't affect anyone.

Reply:
OK, that's what I thought.